### ISSUE TITLE
LitElement static property obj declaration

### ISSUE THREAD

**Issue by emm88n** [JackRobards/lit-analyzer#415]
[there seems to be a proposed implementation on the original repository](https://github.com/runem/web-component-analyzer/pull/283)

problem:
this syntax is not being parsed by the plugin:
```js
// from https://lit.dev/playground/#sample=examples/hello-world
import {html, css, LitElement} from 'lit';

export class SimpleGreeting extends LitElement {
  static styles = css`p { color: blue }`;

  static properties = { // <- it's this syntax, attribute is not detected in syntax completion
    name: {type: String},
  };

  constructor() {
    super();
    this.name = 'Somebody';
  }

  render() {
    return html`<p>Hello, ${this.name}!</p>`;
  }
}
customElements.define('simple-greeting', SimpleGreeting);
```

i.e. , when I try to a syntax completion for the "name" attribute, plugin doesn't show suggestions. but when I try for syntax completion of ".name" property, plugin works.
